src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+| Campaign        | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaign.svg?logo=android&logoColor=white&label=campaign)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaign)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| CampaignClassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+### Manual library import
+
+If you are manually importing the Mobile SDK libraries, update your app dependencies to the latest versions shown in the table above.
+
+### Gradle
+
+If you are using Gradle to manage your app dependencies, the following example shows how to upgrade to latest version in the `build.gradle` file.
+
+```java
+dependencies {
+    //replace sdk-core with Core/Lifecycle/Identity/Signal 2.0
+    //implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+    implementation 'com.adobe.marketing.mobile:core:2.+'
+    implementation 'com.adobe.marketing.mobile:identity:2.+'
+    implementation 'com.adobe.marketing.mobile:lifecycle:2.+'
+    implementation 'com.adobe.marketing.mobile:signal:2.+'
+
+    // replace UserProfile 1.+ with UserProfile 2.0
+    //implementation 'com.adobe.marketing.mobile:userprofile:1.+'
+    implementation 'com.adobe.marketing.mobile:userprofile:2.+'
+
+}
+```
+
+<InlineAlert variant="warning" slots="text"/>
+
+Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/gradle-dependencies.md) for managing Gradle dependencies.
```

Comment:
we now have this file in this repo, let's use relative link here

Reply:
Can you point me to the right page?

Reply:
Not in this repo, it is in the latest version.. I guess we can use a direct link to https://developer.adobe.com/client-sdks/documentation/manage-gradle-dependencies

Reply:
The link shows 404.

Reply:
That link 404s for now since it's not live yet. Since that will be coming in at the same time the ACP repo is made live, we should be fine on that front.